header/partials/subnav/template.html

```diff
@@ -0,0 +1,42 @@
+<div class="o-header__subnav" role="navigation" aria-label="Sub navigation" data-o-header-subnav data-trackable="header-subnav">
+	<div class="o-header__container">
+		<div class="o-header__subnav-wrap-outside">
+			<div class="o-header__subnav-wrap-inside" data-o-header-subnav-wrapper>
+				<div class="o-header__subnav-content">
+
+					<ol class="o-header__subnav-list o-header__subnav-list--breadcrumb" aria-label="Breadcrumb" data-trackable="breadcrumb">
+						{{#each currentNav.ancestors}}
+						<li class="o-header__subnav-item">
+							<a class="o-header__subnav-link" href="{{href}}" data-trackable="{{name}}">
+								{{name}}
+							</a>
+						</li>
+						{{/each}}
+						{{#currentNav}}
+						<li class="o-header__subnav-item">
+							<a class="o-header__subnav-link" href="{{href}}" aria-selected="true" aria-label="Current page">
+								{{name}}
+							</a>
+						</li>
+						{{/currentNav}}
+					</ol>
+
+					{{#if currentNav.children}}
+					<ul class="o-header__subnav-list o-header__subnav-list--children" aria-label="Subsections" data-trackable="subsections">
+						{{#each currentNav.children}}
+						<li class="o-header__subnav-item">
```

Comment:
Do we need the unselected modifier? Surely any default values should just be on the base `o-header__subnav-item` class, no?